src/main/java/core/basesyntax/FieldXY.java

```diff
@@ -13,7 +13,23 @@
 public class FieldXY {
 
     public void moveRobot(Robot robot, int toX, int toY) {
-        //your code
+        int vectorX = toX - robot.getX();
+        int vectorY = toY - robot.getY();
+        Direction vectorDirectionX = vectorX > 0 ? Direction.RIGHT : Direction.LEFT;
+        Direction vectorDirectionY = vectorY > 0 ? Direction.UP : Direction.DOWN;
+        for (int i = 0; i < 5; i++) {
```

Comment:
why do we use `5` here? Can we use `1` or `7`? 
in this case better create constant which will explain meaning of this value
for example: SIZES_OF_THE_WORLD = 4;